campbot/checkers.py

```diff
@@ -486,3 +487,24 @@ def __init__(self):
 
     def test_contribution(self, old_doc, new_doc):
         return old_doc.quality == new_doc.quality
+
+class TitleLevel(object):
+    def __init__(self):
+        self.name = "Titre de niveau trop élevé"
+        self.fail_marker = emoji("/uploads/default/original/2X/0/0178043b1b70e669946f609571bd4b8f7d18e820.png",
+                                 self.name)
+        self.success_marker = ""
+
+    def __call__(self, contrib, old_version, new_version):
+        if old_version is None or new_version is None:
+            return True, True
+
+        old_doc = old_version.document
+        new_doc = new_version.document
+
+        if "redirects_to" in old_doc or "redirects_to" in new_doc:
+            return True, True
+
+        title =  not new_doc.search( r"(^|\n)##[^#]") == None
```

Comment:
A more explicit variable name will help a lot next devs :) 

```suggestion
        has_level2 =  not new_doc.search( r"(^|\n)##[^#]") == None
```